NoSQL/README.md

```diff
@@ -103,6 +109,62 @@ $ docker run --rm -ti --link kvlite:store oracle/nosql:20.3 \
 
 ```
 
+## Using Oracle NoSQL Command-Line from an external host
+
+Start up KVLite in a Docker container. You must give it a name 
+and provide a hostname (resolvable from the external host - an alias to the container host). 
+You need to publish all internal ports.
+
+Startup of KVLite is the default `CMD` of the image:
+
+```shell
+docker run -d --name=kvlite --hostname=kvlite-nosql-container-host --env KV_PROXY_PORT=8080 -p 8080:8080 \
+-p 5000:5000 -p 5010-5020:5010-5020 -p 5021-5049:5021-5049 -p 5999:5999 oracle/nosql:ce
```

Comment:
You can combine the second and third port mappings to `-p 5010-5049:5010-5049` as they are contiguous.

Reply:
I did in this way to document the difference between the two ranges

- 5010-5020 is the KV_HARANGE
- 5021-5049 is the KV_SERVICERANGE

a user can customize those values, the ranges are not always contiguous. And I realized that docker identifies contiguous ports and merges those. When running `docker ps -a`, I have:

`0.0.0.0:5010-5049->5010-5049/tcp`

I will work with our doc team in order to improve this README documentation